docs/reference/module-api/module-overview.md

```diff
@@ -1,6 +1,9 @@
 # Module properties
 
-Module configuration objects may use the following configuration properties.
+Module configuration objects may use the following configuration properties. The overall categories are broadly defined:
+- [Configuration settings](#configuration-settings): Static module settings. These generally cannot use other module settings.
```

Comment:
I still trip over this bit:

are we saying that it can't access settings in other modules? or other settings that exist for other modules.